### PR TITLE
JIRA:VZ-5061 Pipeline fixes and setup for periodic run of upgrade resiliency tests

### PR DESCRIPTION
**Pull request by jmaron99**
# Description

Corrected some issues introduced into upgrade resiliency pipeline and enabled periodic running of the upgrade resiliency test scenarios.  Picked up newer version of VMO with a stricter check for OS readiness.

Fixes VZ-5061

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/verrazzano-bom.json [hunk 1]
```diff
@@ -197,7 +197,7 @@
             },
             {
               "image": "verrazzano-monitoring-operator",
-              "tag": "1.3.0-20220314182600-e8a6f8b",
+              "tag": "1.3.0-20220315201009-44708ce",
```
Comment (bobdonat) [inline platform-operator/verrazzano-bom.json:200]: is this change correct for this branch?

Comment (jmaron99) [reply platform-operator/verrazzano-bom.json:200]: Yep - that's the VMO that has a stricter OS/OSD check.

File: ci/chaos/Jenkinsfile [hunk 2]
```diff
@@ -392,24 +397,48 @@ pipeline {
...
+                            echo "Listing all pods in all namespaces after uninstall"
+                            kubectl get pods --all-namespaces
+                            echo "-----------------------------------------------------"
+
+
+                            # verify Uninstall
+                            ${LOOPING_TEST_SCRIPTS_DIR}/dump_cluster.sh ${WORKSPACE}/verrazzano/build/resources/post-uninstall-resources false
+                            ${LOOPING_TEST_SCRIPTS_DIR}/verify_uninstall.sh ${WORKSPACE}/verrazzano/build/resources
+                        else
+                            echo "Upgrade should still be in progress.  Exiting with error"
```
Comment (pfmackin) [inline ci/chaos/Jenkinsfile:419]: Are we sure upgrade will be in progress

Comment (stevshap) [reply ci/chaos/Jenkinsfile:419]: +1

Comment (jmaron99) [reply ci/chaos/Jenkinsfile:419]: The idea here is that the upgrade has had an error introduced and we want to uninstall the VZ instance that is failing to upgrade.  So the check here is to make sure that the instance isn't in a state other than UpgradeStarted, or in the process of upgrading when we attempt the uninstallation.

**Review by pfmackin — APPROVED** [review]

**Review by stevshap — APPROVED** [review]

**Pull request merged**